### ISSUE TITLE
Unable to create prefix since wine 6.10 with tkg patches

### ISSUE THREAD

**Issue by bear-bulloff** [Frogging-Family/wine-tkg-git#430]
Unable to create prefix since wine 6.10 with tkg patches
![Uploading -4zQ9ohR2Mk.jpg…]()
. Proton ge, wine-staging-tkg have been tested on different distributions, garuda, manjaro, mint. Vanilla wine-staging 6.10-11 works fine. Others, several people who tested, did not face the same problem as mine. The result is in the screenshot.

**Comment by Tk-Glitch** [comment]
The screenshot seemingly failed to upload.

That being said, see https://github.com/Frogging-Family/wine-tkg-git/issues/396

**Comment by bear-bulloff** [comment]
![-4zQ9ohR2Mk](https://user-images.githubusercontent.com/59369672/123260234-02483380-d4fe-11eb-9072-eab98a1ebb42.jpg)

**Comment by bear-bulloff** [comment]
> Скриншот, по-видимому, не удалось загрузить.
> 
> Как бы то ни было, см. [№ 396](https://github.com/Frogging-Family/wine-tkg-git/issues/396)

I have seen that. however, I don't have gst-editing-services installed. installing/removing this plugin does not solve the problem. the problem persists.

**Comment by bear-bulloff** [comment]
I have seen that. however, I don't have gst-editing-services installed. installing/removing this plugin does not solve the problem. the problem persists.

**Comment by Tk-Glitch** [comment]
I cannot reproduce the issue.

**Comment by bear-bulloff** [comment]
> Я не могу воспроизвести проблему.

Could it be related to using an old q8200 processor?

**Comment by Tk-Glitch** [comment]
Actually.. Maybe. I'll investigate.

**Comment by Tk-Glitch** [comment]
Did you use the prebuilt in releases, or did you build it yourself?

**Comment by bear-bulloff** [comment]
I've tried everything. Both ready-made and assembled several times. The result is the same.

**Comment by bear-bulloff** [comment]
> Вы использовали готовые версии или создали их сами?


Here is the log from the steam. Maybe it will be more useful.
[steam-288160.log](https://github.com/Frogging-Family/wine-tkg-git/files/6718213/steam-288160.log)

**Comment by bear-bulloff** [comment]
Here is the log with proton 6.3.5. Everything works well.
[steam-288160.log](https://github.com/Frogging-Family/wine-tkg-git/files/6718242/steam-288160.log)

**Comment by bear-bulloff** [comment]
[steam-288160.log](https://github.com/Frogging-Family/wine-tkg-git/files/6718949/steam-288160.log)

I checked with 6.8 proton-tkg. The prefix is being created.

**Comment by bear-bulloff** [comment]
I checked with 6.11-r4 proton-tkg. The problem remains.
[6.11.r4.steam-288160.log](https://github.com/Frogging-Family/wine-tkg-git/files/6718952/6.11.r4.steam-288160.log)

**Comment by Tk-Glitch** [comment]
That's actually quite different. Prefix creation succeeds in your last log. The game then fails for whatever reason, which seems to be an upstream issue.

**Comment by bear-bulloff** [comment]
> На самом деле это совсем другое. Создание префикса успешно выполняется в вашем последнем журнале. Затем игра терпит неудачу по какой-либо причине, что, по-видимому, является проблемой восходящего потока.

You are right with 6.11-r4, the prefix was created in contrast to the previous ones. What was the problem?
![Снимок экрана_2021-06-26_09-33-18](https://user-images.githubusercontent.com/59369672/123504388-a39bca80-d661-11eb-9a68-b8ce88340781.png)

**Comment by bear-bulloff** [comment]
> На самом деле это совсем другое. Создание префикса успешно выполняется в вашем последнем журнале. Затем игра терпит неудачу по какой-либо причине, что, по-видимому, является проблемой восходящего потока.

With 6.11-r4 a lot of problems, a lot of games do not start or hang at startup. Also I can't add winegstremer library via winecfg.
![Screenshot_SETTINGS : fish_2](https://user-images.githubusercontent.com/59369672/123508446-36476400-d678-11eb-8db8-3e857febc94e.png)

**Comment by tannisroot** [comment]
Remove `gst-editing-services` and the original prefix named .wine located in your home directory, and then try again.

**Comment by bear-bulloff** [comment]
> Удалите `gst-editing-services`и исходный префикс с именем .вино находится в вашем домашнем каталоге, а затем повторите попытку.

gst-editing-services не установлен. В системе установлен wine-staging 6.11 с ним таких проблем нет. 
![Screenshot_113](https://user-images.githubusercontent.com/59369672/123524871-c071e580-d6d5-11eb-9f29-d43f38a86f07.png)

**Comment by bear-bulloff** [comment]
> На самом деле это совсем другое. Создание префикса успешно выполняется в вашем последнем журнале. Затем игра терпит неудачу по какой-либо причине, что, по-видимому, является проблемой восходящего потока.


If I understood correctly from other posts, the problem was with the QPC patch. I managed to successfully build wine-staging 6.11, the prefix is being created, most of the games from my library are working. The question on this topic can be considered closed.

**Comment by tannisroot** [comment]
It's not the QPC patch. This problem also occured with normal staging builds that weren't even made with TKG's utils.

**Comment by bear-bulloff** [comment]
> Это не патч QPC. Эта проблема также возникла с обычными промежуточными сборками, которые даже не были сделаны с utils TKG.

Most likely there are other problems. It helped me to exclude this patch when building wine. On my system, the prefix is now created and the games are launched.

**Comment by Tk-Glitch** [comment]
The QPC patch has been reportedly breaking wine on machines with unstable hardware clocks, so yes, it can be problematic on its own.
